encounters/views.py

```
@@ -1,11 +1,24 @@
 from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.exceptions import PermissionDenied
 from .models import Encounter
 from .serializers import EncounterSerializer
 
 
 class EncounterViewSet(viewsets.ModelViewSet):
     queryset = Encounter.objects.all().order_by('-occurred_at')
     serializer_class = EncounterSerializer
+    permission_classes = [IsAuthenticated]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)